src/data/markdown/docs/03 cloud/07 Cloud FAQ/01 General.md

```diff
@@ -160,7 +160,7 @@ If you are running a `k6 cloud` test, you will be utilizing k6's cloud infrastru
 
 To open your firewall to k6 cloud traffic, you have multiple options.
 
-1 - Open up your firewall to the whole range of AWS IP addresses used by the load zones where you want to run your load test from. We list [here](/cloud/cloud-faq/what-ip-addresses-are-used-by-the-k6-cloud) the full range of IP addresses.
+1 - Open up your firewall to the whole range of AWS IP addresses used by the load zones where you want to run your load test from. We list [here](/cloud/cloud-faq/general-questions/#how-to-open-a-firewall-to-k6-cloud-service-for-cloud-tests) the full range of IP addresses.
```

Comment:
This just links back to this section. 

I think you just needed to add `general-questions/#` before the current `what-ip...` ?